cpp/src/decisiontree/batched-levelalgo/kernels.cuh

```diff
@@ -655,7 +708,7 @@ __global__ void computeSplitRegressionKernel(
   __syncthreads();
 
   /* Make a second pass over the data to compute gain */
-
+  auto coloffset = col * input.M;
```

Comment:
Is `coloffset` used anywhere in the kernel?

Reply:
seems to be used in [L716](https://github.com/rapidsai/cuml/blob/d6b297604b1abaeeceb273fc72e201594641b8a6/cpp/src/decisiontree/batched-levelalgo/kernels.cuh#L716) and [L729](https://github.com/rapidsai/cuml/blob/d6b297604b1abaeeceb273fc72e201594641b8a6/cpp/src/decisiontree/batched-levelalgo/kernels.cuh#L729)